src/System.IO/src/System.IO.csproj

```diff
@@ -76,9 +82,14 @@
     <TargetingPackReference Include="System" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' != 'netstandard13aot'">
     <None Include="project.json" />
   </ItemGroup>
+  
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard13aot'">
+    <TargetingPackReference Include="System.Private.Interop" />
+    <None Include="netstandard13aot\project.json" />
```

Comment:
Can you add a comment here that says you needed to fork the dependency group for AOT vs non-AOT but could not due to https://github.com/dotnet/cli/issues/465 & https://github.com/nuget/home/issues/1660